### DISCUSSION TITLE
2024-02-27 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#299] [Meetings]
### Date: 2024-02-27, 4:00 – 5:00 pm EST
**Chair(s): Marjorie Rallins**

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
| 4:00 - 4:05 | Welcome & opening remarks | Marjorie Rallins |
| 4:05 - 4:15 | USCDI v5 input|Marjorie Rallins|
|4:15-4:45|Conformance and Validation (cont.)|Eza Hafeza, All|
| 4:45-4:55| New and Open Proposals (as time permits) | Eza Hafeza, All |
| 4:55 - 5:00 | Topics for Next Meeting | All
||Demo a request using GitHub||
||Input into attributes/properties for new units (file linked)||
| 5:00 | Adjourn | Eza Hafeza |

***OPEN ACTION ITEMS***
|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
| ----------- | -----| -----| -----|
| Continue asynchronous discussion on conformance / validation, [UCUM validation #250](https://github.com/orgs/ucum-org/discussions/250) | 2023-11-28 | Post first release | All |
| Advance membership of Jeremy Lieth to next phase | 2023-07-25 | 2023-11-14 | April Lackey |
| Collect use cases / user stories that may help inform license discussions | 2023-07-25 | 2023-11-14 | Tim Briscoe et. al. |

***LINKS TO REFERENCE MATERIALS***
[UCUM Committee_Meeting Summary_2024-01-23_FINAL](https://docs.google.com/document/d/1cNXB7_f1vVvVF93oZWUmw5c_hmjNtoTg/edit?usp=drive_link&ouid=110875911742539584915&rtpof=true&sd=true)
[UCUM New Unit Template_2024-01](https://docs.google.com/spreadsheets/d/1xBGaNLuMaJBFnFdi81ezDF5oHzN_Ozww/edit#gid=1763020506)

@ucum-org/regenstrief 
@ucum-org/advisors

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release. 

Thank you,

Best Regards,
The LOINC Team
